src/python/tests/core/bot/fuzzers/utils_test.py

```diff
@@ -12,73 +12,88 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Tests for fuzzers.utils."""
-from pyfakefs import fake_filesystem_unittest
+
+import os
+import shutil
+import tempfile
+import unittest
 
 from bot.fuzzers import utils
 from system import environment
 from tests.test_libs import helpers as test_helpers
-from tests.test_libs import test_utils
 
 
-class IsFuzzTargetLocalTest(fake_filesystem_unittest.TestCase):
+class IsFuzzTargetLocalTest(unittest.TestCase):
   """is_fuzz_target_local tests."""
 
   def setUp(self):
     test_helpers.patch_environ(self)
-    test_utils.set_up_pyfakefs(self)
+    self.temp_dir = tempfile.mkdtemp()
+
+  def tearDown(self):
+    shutil.rmtree(self.temp_dir, ignore_errors=True)
+
+  def _create_file(self, name, contents=b''):
+    path = os.path.join(self.temp_dir, name)
+    with open(path, 'wb') as f:
+      f.write(contents)
+
+    return path
 
   def test_not_a_fuzzer_invalid_name(self):
```

Comment:
Can you add a test where a file handle is passed (and not a string path)?

Reply:
Done!